msteams-platform/task-modules-and-cards/cards/cards-reference.md

```diff
@@ -126,8 +126,9 @@ The following table provides the features that support Adaptive Cards:
 > * Teams platform supports v1.5 or earlier of Adaptive Card features for bot sent cards and action based message extensions.
 > * Teams platform supports v1.5 or earlier of Adaptive Card features for other capabilities, such as cards sent by user (search based message extensions and link unfurling), tabs, and task modules.
 > * Positive or destructive action styling is not supported in Adaptive Cards on the Teams platform.
-> * Media elements are currently not supported in Adaptive Card on the Teams platform.
+> * Media elements are not supported in Adaptive Card on the Teams platform.
 > * Test your full width Adaptive Card in narrow form factors such as mobile and meeting side panels to ensure that content is not truncated.
+> * Adaptive Cards in Teams don't support file or image upload capability.
```

Comment:
It's not a capability.

Reply:
Updated as Adaptive Cards in Teams don't support file or image upload.